core/players-handbook/classes/class-druid.xml

```diff
@@ -614,11 +614,11 @@
 		</sheet>
 		<rules>
 			<stat name="circle forms:cr" value="1" />
-			<stat name="circle forms:cr" value="2" level="6"/>
-			<stat name="circle forms:cr" value="3" level="9"/>
-			<stat name="circle forms:cr" value="4" level="12"/>
-			<stat name="circle forms:cr" value="5" level="15"/>
-			<stat name="circle forms:cr" value="6" level="18"/>
+			<stat name="circle forms:cr" value="1" level="6"/>
+			<stat name="circle forms:cr" value="1" level="9"/>
+			<stat name="circle forms:cr" value="1" level="12"/>
+			<stat name="circle forms:cr" value="1" level="15"/>
+			<stat name="circle forms:cr" value="1" level="18"/>
```

Comment:
put in a stacking bonus `bonus="base"` on the stats instead, thanks

Reply:
sure

Reply:
done